pyquaternion/quaternion.py

```diff
@@ -156,7 +157,7 @@ def _validate_number_sequence(self, seq, n):
 
     # Initialise from matrix
     @classmethod
-    def _from_matrix(cls, matrix):
+    def _from_matrix(cls, matrix, **kwargs):
```

Comment:
For clarity of intent, I think it would be better to explicitly define the tolerance parameters with defaults here:
`def _from_matrix(cls, matrix, rtol=1e-05, atol=1e-08):`
Then below, instead of unpacking kwargs, pass them through to `np.isclose()` and `np.allclose()` explicitly.

Reply:
Done.